apps/ledger-live-desktop/src/renderer/families/cosmos/DelegationFlowModal/Info/index.jsx

```diff
@@ -39,11 +39,13 @@ export default function CosmosEarnRewardsInfoModal({ name, account, parentAccoun
     <EarnRewardsInfoModal
       name={name}
       onNext={onNext}
-      description={t("cosmos.delegation.flow.steps.starter.description")}
+      description={t("cosmos.delegation.flow.steps.starter.description", {
+        currencyTicker: account.currency.ticker,
+      })}
       bullets={[
         t("cosmos.delegation.flow.steps.starter.bullet.0"),
         t("cosmos.delegation.flow.steps.starter.bullet.1"),
-        t("cosmos.delegation.flow.steps.starter.bullet.2"),
+        t("cosmos.delegation.flow.steps.starter.bullet.2", { numberOfDays: 21 }),
```

Comment:
Is it 21 for all cosmos chains? No need to get that value dynamically?
(same comment for the other files)

Reply:
Well this part is kind of confusing yet, I'm not sure how we will handle this and how to get this information from the blockchain. I add 21 as placeholder because it's by default BUT osmosis is 14 for exemple

Reply:
I'll take care of this parameter later. It varies according to different currency.